gpt_json/types_oai.py

```diff
@@ -0,0 +1,60 @@
+from typing import Dict, List, Optional, Union
```

Comment:
where best

Reply:
The current project target is > 3.10, so native typing should be supported. Instead of `List/Dict` you can just use `list/dict`, and Optional can just be `Value | None`.

Reply:
ah I meant the entire contents of the file